port_ocean/core/defaults/initialize.py

```diff
@@ -50,10 +53,47 @@ def deconstruct_blueprints_to_creation_steps(
     )
 
 
+async def _initialize_required_integration_settings(
+    port_client: PortClient,
+    default_mapping: PortAppConfig,
+    integration_config: IntegrationConfiguration,
+) -> None:
+    try:
+        integration = await port_client.get_current_integration(should_log=False)
+        if not integration:
+            await port_client.create_integration(
+                integration_config.integration.type,
+                integration_config.event_listener.to_request(),
+                port_app_config=default_mapping,
+            )
+        elif not integration["config"]:
+            await port_client.patch_integration(
+                integration_config.integration.type,
+                integration_config.event_listener.to_request(),
+                port_app_config=default_mapping,
+            )
+    except httpx.HTTPStatusError as err:
+        raise Exception(f"Failed to apply default mapping: {err.response.text}.")
+
+    logger.info("Checking for diff in integration configuration")
+    changelog_destination = integration_config.event_listener.to_request().get(
+        "changelog_destination"
+    )
+    if (
+        integration["changelogDestination"] != changelog_destination
+        or integration["installationAppType"] != integration_config.integration.type
+        or integration.get("version") != port_client.integration_version
+    ):
+        await port_client.patch_integration(
+            integration_config.integration.type,
+            changelog_destination,
+            default_mapping,
+        )
+
+
 async def _create_resources(
     port_client: PortClient,
     defaults: Defaults,
-    integration_config: IntegrationConfiguration,
 ) -> None:
```

Comment:
What about trying to create some of the missing blueprints?

Reply:
is that in scope of this issue?

Reply:
Please talk to @MPTG94

Reply:
he said that in any case if one or more of the blueprints that the integration needs to create in its initialization already exist, he expects the integration to not create any blueprints (just keep the state as it is), so no need for that for now at least